test/e2e/e2e_suite_test.go

```diff
@@ -223,38 +247,60 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 			configPath,
 			clusterctlConfigPath,
 			bootstrapClusterProxy.GetKubeconfigPath(),
-			strings.Join(ipamLabelsRaw, ";"),
+			strings.Join(ipClaimLabelsRaw, ";"),
 		}, ","),
 	)
 }, func(data []byte) {
 	// Before each ParallelNode.
+
 	parts := strings.Split(string(data), ",")
 	Expect(parts).To(HaveLen(5))
 
 	artifactFolder = parts[0]
 	configPath = parts[1]
 	clusterctlConfigPath = parts[2]
 	kubeconfigPath := parts[3]
-	ipamLabelsRaw := parts[4]
+	ipClaimLabelsRaw := parts[4]
 
 	namespaces = map[*corev1.Namespace]context.CancelFunc{}
 
-	By("Initializing the vSphere session to ensure credentials are working", initVSphereSession)
+	if testTarget == VCenterTestTarget {
```

Comment:
You have to pass through the `testTarget` variable via the strings.Join / split above, otherwise the ginkgo workers/nodes do not get the value when running tests in-parallel!